.github/workflows/docker-publish.yaml

```diff
@@ -3,17 +3,19 @@ name: Docker Publish
 on:
   push:
     branches:
-      - 'master'
```

Comment:
Why did we remove master or are we saying we will just use images from develop instead ie. for prod deployments?

Reply:
That's not part of this PR but incidental due to the git structure of it. I removed it as we need two different image tags (to ensure we can run both develop and prod in parallel) and therefore need to overhaul that anyways.